### PR TITLE
babe wake up, new roach just dropped - benzin roach

### PR DESCRIPTION
**Pull request by bearrrrrrrr**
## About The Pull Request

[dis](https://github.com/discordia-space/CEV-Eris/pull/7163)

additonally: cooking the benzin meat explodes it. used this same object to re-add phoron being smelted into an explosion

## Why It's Good For The Game

"They explode when shot, which could be either extra danger or used to player advantage.
Rarity between seuche and panzer roaches."

## Changelog
```changelog
add: benzin roach
```

### CODE REVIEW

**Comment by bearrrrrrrr** [comment]
emojiocracy is 7 yes / 0 no / 8 'yes but make the meat kill you when you cook it'

File: code/modules/mob/living/carbon/superior_animal/roach/types/benzin.dm [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+/mob/living/carbon/superior_animal/roach/benzin/bullet_act(obj/item/projectile/P, def_zone)
+	. = ..()
+	if(prob(80))
+		explosion(get_turf(src), -1, -1, 2, 3)
+
+
+/mob/living/carbon/superior_animal/roach/benzin/attackby(obj/item/I, mob/living/user, params)
+	if(user.a_intent == I_HELP && istool(I))
+		var/obj/item/tool/T = I
+		if(T.use_fuel_cost)
```
Comment (fludd12) [inline code/modules/mob/living/carbon/superior_animal/roach/types/benzin.dm:36]: Does this actually let you refuel the tools? My brain isn't working right lately so I can't track this logic. If so, I love it.

**Review by fludd12 — APPROVED** [review]
Looks _great_ to me. I'm excited for when a den spawns like 30 of these in one room and someone has an itchy trigger finger.

File: code/modules/mob/living/carbon/superior_animal/roach/types/benzin.dm (hunk 1 above)
Comment (fludd12) [reply code/modules/mob/living/carbon/superior_animal/roach/types/benzin.dm:36]: Holy shit it does.

Comment (bearrrrrrrr) [reply code/modules/mob/living/carbon/superior_animal/roach/types/benzin.dm:36]: Y U P

Comment (bearrrrrrrr) [reply code/modules/mob/living/carbon/superior_animal/roach/types/benzin.dm:36]: it also detonates when you melee them w/ a lit welder i'm fairly certain

**Pull request merged**